Library/Homebrew/extend/os/mac/keg_relocate.rb

```diff
@@ -24,7 +24,7 @@ def relocate_dynamic_linkage(relocation)
       file.ensure_writable do
         if file.dylib?
           id = dylib_id_for(file).sub(relocation.old_prefix, relocation.new_prefix)
-          change_dylib_id(id, file)
+          file.change_dylib_id(id)
```

Comment:
The `change_dylib_id` still accepts 2 arguments. Shouldn't this be  `file.change_dylib_id(id, file)`?

Reply:
Whoops, that was a mistake. It should only take 1, I'll fix that up now.